Editor/TreasuredMapEditor.cs

```diff
@@ -466,13 +466,73 @@ public override void OnInspectorGUI()
                                                             @"[a-zA-Z0-9\-]").Success))
                 {
                     if (GUILayout.Button(
-                            EditorGUIUtility.TrTextContentWithIcon("Export",
+                            EditorGUIUtility.TrTextContentWithIcon(_map.exportSettings.ExportType == ExportType.Export ? "Export" : "Production Export",
                                 $"Export scene to {TreasuredSDKPreferences.Instance.customExportFolder}/{_map.exportSettings.folderName}",
-                                "SceneLoadIn"), Styles.exportButton, GUILayout.MaxWidth(150)))
+                                "SceneLoadIn"), Styles.exportButton, _map.exportSettings.ExportType == ExportType.ProductionExport ? GUILayout.MaxWidth(200) : GUILayout.MaxWidth(150)))
                     {
                         try
                         {
                             Exporter.Export(_map);
+
+                            var exporters = ReflectionUtils.GetSerializedFieldValuesOfType<Exporter>(_map);
```

Comment:
Exporter can be access through map object, using reflection is not required.

Reply:
Removed reflection.